### PR TITLE
[CAR-207] Scan history and fixes

### PR DESCRIPTION
**Pull request by KariGarcia**

### CODE REVIEW

File: common/constants.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const BASE_URL =
-  process.env.NODE_ENV === 'production' ? 'https://actnow.wmdd4950.com' : 'http://localhost:3000';
+  process.env.NODE_ENV !== 'production' ? 'https://actnow.wmdd4950.com' : 'http://localhost:3000';
```
Comment (luferupa) [inline common/constants.ts]: Should we change it? 😅

File: navigation/index.tsx [hunk 2]
```diff
@@ -68,8 +69,9 @@ function RootNavigator() {
     <Stack.Navigator>
       <Stack.Screen name="Root" component={BottomTabNavigator} options={{ headerShown: false }} />
       <Stack.Screen name="NotFound" component={NotFoundScreen} options={{ title: 'Oops!' }} />
```
Comment (luferupa) [inline navigation/index.tsx:71]: Is this the one we should show if the result has an error?

**Review by luferupa — APPROVED** [review]
Added small comments

**Comment by luferupa** [comment]
Added some fixes:
- Scanner endpoints errors control
- Points added for Text Search
- Scan history scrollable

File: common/constants.ts (hunk 1 above)
Comment (KariGarcia) [reply common/constants.ts]: yessss

**Pull request merged**